applications/StructuralMechanicsApplication/python_scripts/structural_response.py

```diff
@@ -191,15 +191,35 @@ class MassResponseFunction(ResponseFunctionBase):
 
     def __init__(self, identifier, response_settings, model):
         self.identifier = identifier
+
+        default_settings = Parameters("""{
+            "response_type"            : "mass",
+            "material_import_settings" : {
+                "materials_filename"   : "SPECIFY_MATERIALS_FILE"
+            },
+            "model_part_name"          : "SPECIFY_MODELPART_NAME",
+            "domain_size"              : 3,
```

Comment:
also this should not have a default, it can be dangerous 
=> for this reason it is -1 by default in Fluid & Structure